package.json

```diff
@@ -24,6 +24,7 @@
         "@mui/icons-material": "^5.11.16",
         "@mui/material": "^5.13.6",
         "@pmmmwh/react-refresh-webpack-plugin": "^0.5.10",
+        "@types/node": "^22.10.0",
```

Comment:
This should probably be a dev dependency

Reply:
Why is that? I've installed the rpm version, which i never got working because of the misformated path. I am by far not a typescript expert, but should'nt the module be loaded for production builds as well, if it's used?

Reply:
The @types modules are just type hints used by the developer. As typescript transpiles down to js, there are no types anyway.

Basically, the module has no code that would actually be "compiled".

I'd suggest moving it to a dev dependency and then running it locally to verify it works.

Reply:
Granted this is an electron app and I have no idea what I'm doing when it comes to electron anyway do I could be wrong 😆